tests/Unit/Console/Command/LogCommandTest.php

```diff
@@ -146,7 +152,8 @@ public function testPaginate()
         $input = new ArrayInput([], $this->command->getDefinition());
         $output = $this->output;
 
-        $this->application->setTerminalDimensions(100, 14);
+        $this->setTerminalDimensions(100, 14);
```

Comment:
might be better to keep this at the application level (see failing tests)?

Reply:
You mean, move this to `PhpBench\Console\Application`?

Reply:
I guess the question is why did we change this? Did it recently become part of the command class or? Can we keep it as a method call on the `application` ?

Reply:
The method was removed in `symfony/console`, that's why we either need to stop using it, or implement it ourselves.

Reply:
It's replaced by the [Terminal](https://symfony.com/blog/new-in-symfony-3-2-console-improvements-part-2#introduced-a-new-terminal-class) class.

But if we can replace with env vars, that's probably the best solution :+1: